proto/pdpb.proto

```diff
@@ -498,6 +500,7 @@ message GetGCSafePointResponse {
     ResponseHeader header = 1;
 
     uint64 safe_point = 2;
+    uint64 min_safe_point_limit = 3;
```

Comment:
I think the returned `safe_point` is always the minimum value, so `min_safe_point_limit ` is not necessary.

Reply:
TiDB need to check how far it can advance the safepoint before it actually advance it, because it need to do `DeleteRange` step with the limited safepoint before sending its safepoint to PD.